libs/platform/src/lib/components/dynamic-page/dynamic-page-content/dynamic-page-content.component.ts

```diff
@@ -0,0 +1,191 @@
+import { CdkScrollable, ScrollDispatcher } from '@angular/cdk/scrolling';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    forwardRef,
+    Input,
+    NgZone,
+    OnDestroy,
+    OnInit,
+    Renderer2,
+    TemplateRef,
+    ViewChild
+} from '@angular/core';
+import { BACKGROUND_TYPE, CLASS_NAME, DYNAMIC_PAGE_CHILD_TOKEN, RESPONSIVE_SIZE } from '../constants';
+import { DynamicPageService } from '../dynamic-page.service';
+import { addClassNameToElement } from '../utils';
+
+@Component({
+    selector: 'fdp-dynamic-page-content',
+    templateUrl: './dynamic-page-content.component.html',
+    styleUrls: ['./dynamic-page-content.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: DYNAMIC_PAGE_CHILD_TOKEN,
+            useExisting: forwardRef(() => DynamicPageContentComponent)
+        }
+    ]
+})
+export class DynamicPageContentComponent extends CdkScrollable implements OnInit, OnDestroy {
+    /**
+     * label for the tab. If label is provided, tab navigation will be internally set up.
+     */
+    @Input()
```

Comment:
Properties alignment has to be done as per angular guidelines.  same as be done where ever applicable. For more details, angular.io will help as well as other components implementations will help

Reply:
followed ngx guidelines and fixed wherever needed.